workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/src/Generation/WorkerGenerationJob.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Generic;
+using System.IO;
+using Improbable.CodeGeneration.FileHandling;
+using Improbable.CodeGeneration.Jobs;
+
+namespace Improbable.Gdk.CodeGenerator
+{
+    public class WorkerGenerationJob : CodegenJob
+    {
+        private readonly string relativeOutputPath = "improbable";
+        private readonly string package = "Improbable";
+        private readonly List<string> workerTypesToGenerate;
+
+
+        private const string fileExtension = ".cs";
+
+        public WorkerGenerationJob(string outputDir, IFileSystem fileSystem, List<string> workerTypes) : base(
+            outputDir, fileSystem)
+        {
+            InputFiles = new List<string>();
+            OutputFiles = new List<string>();
+            
+            workerTypesToGenerate = workerTypes;
+            
+            var workerFileName = Path.ChangeExtension("WorkerMenu", fileExtension);
```

Comment:
Given this is the same in the RunImpl and there will only be one of these - make this a class level variable - less likely to get out of sync if we make changes to the name